### PR TITLE
backpack visual tweaks

### LINKED ISSUE #16462: [Bug]: backpack sprite errors
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.
- [X] The round I am reporting the bug from, or I am going to talk about, has already ended.

### Location

Event / Other

### Describe the issue

Every satchel and pocketbook seems to have some kind of masking issue where the band meets the player sprite's shoulder, including leather satchels and pocketbooks with the band hidden (regular satchels have a slightly different variation on the problem):
maskingerror-leathersatchel
maskingerror-pocketbook
maskingerror-satchel

Other backpack types might be suffering from this too. I suspect this is probably meant to be light shading via masks that got corrupted at some point.

Bonus issue: pocketbooks extend lower than they should on the left-facing sprite:
pocketbook error

### How to reproduce

Fiddle around with satchels and pocketbooks in character setup or ingame.

### Round ID

_No response_

### Staff ckey

_No response_

### PR DESCRIPTION
**Pull request by crunchypretzels**
Removed the clasp from all back items that had one, as far as I could tell. Unpopular enough to warrant it, per opinions of those I asked.

Not technically a bugfix.

fixes #16462

This also adds a few pixels to the Fade - Clean Low hairstyle, to avoid it looking like a hairpiece or something.

### CODE REVIEW

**Comment by crunchypretzels** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]
Should probably seperate the hair change to a different PR

File: icons/obj/storage/duffelbag.dmi
Comment (atteria) [inline icons/obj/storage/duffelbag.dmi:1]: Looks like there's a rogue pixel on `duffel-emt_ba`

**Review by alsoandanswer — COMMENTED** [review]
You need to edit the mask (bright pink strap) in the dmi to fix the masking bug.

File: icons/obj/storage/duffelbag.dmi
Comment (crunchypretzels) [reply icons/obj/storage/duffelbag.dmi:1]: Resolution Forthcoming

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
Should probably get a feedback thread on the clasps

**Comment by crunchypretzels** [comment]
!wip

**Comment by crunchypretzels** [comment]
https://forums.aurorastation.org/topic/18582-clasp-removal-from-satchels-duffels-etc/

**Review by atteria — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
Stale

**Pull request closed without merging**